### PR TITLE
Scala 2.12.11 and 2.13.2

### PR DESCRIPTION
**Pull request by ennru**
* Use the Scala versions used in Kafka core (to keep Embedded Kafka a little bit more #1114)
	* 2.12 https://mvnrepository.com/artifact/org.apache.kafka/kafka_2.12/2.6.0
	* 2.13 https://mvnrepository.com/artifact/org.apache.kafka/kafka_2.13/2.6.0
* Update the Silencer
* Adapt code to avoid new warnings in Scala 2.13.2

**This makes Scala 2.13.2 the default version for all modules.**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -91,7 +91,7 @@ val commonSettings = Def.settings(
   licenses := Seq("Apache-2.0" -> url("http://opensource.org/licenses/Apache-2.0")),
   description := "Alpakka is a Reactive Enterprise Integration library for Java and Scala, based on Reactive Streams and Akka.",
   crossScalaVersions := Seq(Scala212, Scala213),
-  scalaVersion := Scala212,
+  scalaVersion := Scala213,
```
Comment (ignasi35) [inline build.sbt:94]: Sneaky!

**Review by ignasi35 — APPROVED** [review]

**Comment by ennru** [comment]
Oh wow, Kafka hits Scala binary compatibility issues as it looks to me
```
[info]   java.lang.NoClassDefFoundError: scala/math/Ordering$$anon$7
[info]   at kafka.api.ApiVersion$.orderingByVersion(ApiVersion.scala:45)
[info]   at kafka.api.ApiVersion.compare(ApiVersion.scala:141)
[info]   at kafka.api.ApiVersion.compare$(ApiVersion.scala:140)
[info]   at kafka.api.KAFKA_2_6_IV0$.compare(ApiVersion.scala:348)
[info]   at kafka.api.KAFKA_2_6_IV0$.compare(ApiVersion.scala:348)
[info]   at scala.math.Ordered.$greater$eq(Ordered.scala:91)

```
https://travis-ci.org/github/akka/alpakka-kafka/jobs/733600592#L654

**Comment by ennru** [comment]
The error occurs in EmbeddedKafka on Scala 2.12.12 (Scala 2.13.3 works)
https://travis-ci.org/github/akka/alpakka-kafka/jobs/734533513#L443

**Comment by ennru** [comment]
The Kafka broker itself is compiled against Scala 2.12.11 (https://mvnrepository.com/artifact/org.apache.kafka/kafka_2.12/2.6.0).

**Comment by lrytz** [comment]
Seems there is a broken assumption here

> Inline more aggressively when compiling the `core` jar since it's not meant to be used as a library [...] This is only safe if the Scala library version is the same at compile time and runtime

```gradle
    if (project.name.equals('core'))
      inlineFrom = ["-opt-inline-from:scala.**", "-opt-inline-from:kafka.**", "-opt-inline-from:org.apache.kafka.**"]
```

https://github.com/apache/kafka/blob/2.6.0/build.gradle#L485-L492

**Comment by ennru** [comment]
Rolled back this PR to Scala 2.12.11 and 2.13.2 so we can continue dropping Embedded Kafka in #1209

**Pull request merged**